README.md

```diff
@@ -43,7 +43,7 @@ Verification (rather than stub it) all non-relative urls will cause the handover
     "businessVerificationCheck" : true,
     "continueUrl" : "/test",
     "deskProServiceId" : "abc",
-    "optServiceName" : "Service Name",
+    "optServiceName" : "Service Name", // deprecated, use labels.en.optServiceName
```

Comment:
Can not see "en.optServiceName" in labels object. Have you got the latest version?